We have next logging message:2024-01-05 19:52:00,028:ERROR -Exception
Traceback (most recent call last):
  File "D:\Python\Python_ItStep\lesson_8.py", line 4, in <module>
    print(10/0)
          ~~^~
ZeroDivisionError: division by zero
